src/main/resources/fxml/stats.fxml

```diff
@@ -72,4 +72,24 @@
 		<DataLabel byteFormat="%stats.encr.total.data.none" kibFormat="%stats.encr.total.data.kib" mibFormat="%stats.encr.total.data.mib" gibFormat="%stats.encr.total.data.gib" dataInBytes="${controller.totalBytesEncrypted}"/>
 		<FormattedLabel format="%stats.write.accessCount" arg1="${controller.filesWritten}"/>
 	</VBox>
+	<!-- Access -->
+	<VBox prefWidth="300" prefHeight="300" spacing="6" alignment="CENTER">
+		<FormattedLabel styleClass="label-large" format="%stats.access.accessCount" arg1="${controller.filesAccessed}"/>
+		<AreaChart fx:id="accessChart" styleClass="io-stats" createSymbols="false" animated="false">
+			<xAxis>
+				<NumberAxis fx:id="accessChartXAxis" styleClass="io-stats" autoRanging="false" forceZeroInRange="false" side="BOTTOM"/>
+			</xAxis>
+			<yAxis>
+				<NumberAxis fx:id="accessChartYAxis" styleClass="io-stats" autoRanging="true" forceZeroInRange="true" side="LEFT" tickUnit="Infinity"/>
+			</yAxis>
+			<cursor>
+				<Cursor fx:constant="DEFAULT"/>
+			</cursor>
+		</AreaChart>
+		<Label></Label> <!-- So the three graphs are aligned -->
```

Comment:
```suggestion
		<Label/> <!-- So the three graphs are aligned -->
```

Reply:
Fixed in https://github.com/cryptomator/cryptomator/pull/2233/commits/e097bea1f55eb12feeb01f7a8001f2bad72a0cbb